modules/tutorials/examples/logging-aggregator/main.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+echo "Installing ZooKeeper Operator"
+# tag::zk-op[]
+stackablectl release install -i secret -i commons -i zookeeper 23.7
+# end::zk-op[]
+
+# tag::vector-agg[]
+helm install -f vector-aggregator-values.yaml vector-aggregator vector/vector
+# end::vector-agg[]
+
+# tag::vector-discovery[]
+kubectl apply -f vector-aggregator-discovery.yaml
+# end::vector-discovery[]
+
+# tag::zk[]
+kubectl apply -f zookeeper.yaml
+# end::zk[]
+
+# TODO make this somehow a testable thing
+# maybe get the logs from the agg. and grep for a specific line?
```

Comment:
If you wait until the ZooKeeper instances are ready then this script is a working test.

```suggestion
kubectl rollout status statefulset simple-zk-server-default --timeout=5m
kubectl wait \
    --for=jsonpath='.status.readyReplicas'=3 \
    --timeout=5m \
    statefulsets.apps/simple-zk-server-default
```

Reply:
@siegfriedweber Why the "rollout status" and the "kubectl wait", shouldn't one of them be enough?

Reply:
`kubectl rollout status …` waits until the StatefulSet is deployed and the pods are running and `kubectl wait …` waits until   the pods are ready. If `kubectl rollout status …` is left out then `kubectl wait …` fails immediately because it cannot find the given StatefulSet. If `kubectl wait …` is left out then the pods are running but not necessarily ready.